tests/conftest.py

```diff
@@ -1736,3 +1736,11 @@ def store_with_multiple_cases_and_samples(
         helpers.add_case_with_sample(base_store=store, case_id=case_id, sample_id=sample_id)
 
     yield store
+
+
+@pytest.fixture(name="store_with_panels")
```

Comment:
```Python
@pytest.fixture(name="store_with_two_panels")
```

Reply:
I added another panel to the store, the point is that the store contains multiple panels so I think the naming is ok as is.